omero/sysadmins/unix/install-web/install-nginx.txt

```diff
@@ -13,18 +8,94 @@ Nginx Gunicorn configuration (Unix/Linux)
       see :ref:`python-requirements` on the
       :doc:`/sysadmins/version-requirements` page.
 
-Install `Django 1.8`_ and `Gunicorn <http://docs.gunicorn.org/>`_ using
-package requirements file:
+
+OMERO.web uses
+`Web Server Gateway Interface (WSGI) <http://wsgi.readthedocs.org/en/latest/learn.html>`_
+as a primary deployment platform, the Python standard
+`PEP 3333 <https://www.python.org/dev/peps/pep-3333/>`_
+that describes a web server communicates with web applications.
+OMERO.web deployment can be configured with synch and async workers.
+Sync workers are faster and recomanded for a data reposority with
+:ref:`download_restrictions`. If you wish to offer users ability
+to downlaod data you have to user async workers, read more about
```

Comment:
downlaod -> download
user -> use?

Also would read better as "If you wish to offer users *the* ability to…"